lib/features/base/base_controller.dart

```diff
@@ -375,4 +375,10 @@ abstract class BaseController extends GetxController
     authorizationInterceptors.clear();
     await cachingManager.closeHive();
   }
+
+  @override
+  void onClose() {
+    super.onClose();
+    dispose();
```

Comment:
Please test again on web

Reply:
The only place we have concern about `dispose()` callback is `ComposerController`, so it's fine

Reply:
Many controllers inherit from it. Are you sure other controllers are not affected?. Please test the features on the website: DestinationPicker, Signature, Forward Email, Vacation, etc.

Reply:
We only care about the one overriding `dispose`. If originally `dispose` "does nothing" (as its author said), why do we have to care if its not modified?

Reply:
> We only care about the one overriding `dispose`. If originally `dispose` "does nothing" (as its author said), why do we have to care if its not modified?

The problem here is that we handle many things in the `dispose` function. In the current version, if `dispose` is not called, the handlers will not be executed. And now if you call it in `onClose` maybe some of those actions will throw an error, we don't know. So we need to check that `controllers` inherit from `BaseController`.

Reply:
Please ensure that you have tested most of the app's features and that they are working properly with this change.